types/w3c-web-serial/w3c-web-serial-tests.ts

```diff
@@ -190,3 +190,15 @@ async function example_7c() {
         }
     }, 1_000);
 }
+
+/*~ https://wicg.github.io/serial/#example-8 */
+
+async function example_8() {
+    // Request a serial port.
+    const port = await navigator.serial.requestPort();
+
+    // Then later... revoke permission to the serial port.
+    if (port.forget) {
```

Comment:
Please remove this conditional as well since it doesn't match the example from the specification.

Reply:
done.